src/components/Calculator.js

```diff
@@ -1,113 +1,220 @@
+  }
+
   return (
     <div>
       <table>
-        <Display />
+        <Display total={String(disp)} />
         <tbody>
           <tr>
             <td>
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="AC"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 AC
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="+/-"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 +/-
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="%"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 %
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-orange">
+              <button
+                type="button"
+                value="÷"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-orange"
+              >
                 ÷
               </button>
             </td>
           </tr>
           <tr>
             <td>
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="7"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 7
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="8"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 8
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="9"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 9
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-orange">
+              <button
+                type="button"
+                value="*"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-orange"
+              >
```

Comment:
- [ ] @rushemaa great work implementing the changes requested however when I try to multiply two numbers I get the error below, kindly check your multiplication symbol
![image](https://user-images.githubusercontent.com/61408860/235089060-31355ca9-b685-44de-bfa9-0305837154a9.png)
![image](https://user-images.githubusercontent.com/61408860/235089140-d9b8b4b1-0a8f-4902-94d5-da93fb175ac8.png)